openingd/openingd.c

```diff
@@ -1073,6 +1073,7 @@ static u8 *fundee_channel(struct state *state, const u8 *open_channel_msg)
 		u8 *errmsg = towire_errorfmt(NULL, &state->channel_id,
 					     "%s", err_reason);
 		sync_crypto_write(state->pps, take(errmsg));
+		tal_free(err_reason);
```

Comment:
There is another leak here, so I have a larger change with #3298 which includes this.

Not that it's wrong, so I can merge after this if you want.

Reply:
No problem, I'll merge yours and then rebase this. It's marked as a draft so I don't create too many cleanup PRs and bundle them on friday :-)